app/src/main/res/values/strings.xml

```diff
@@ -27,6 +27,7 @@
 
     <!--profile details-->
     <string name="user_name">Username</string>
+    <string name="e_mail">E-Mail</string>
```

Comment:
just `Email` is fine there is already a string for that. You can use it

Reply:
Corrected.